sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -81,4 +81,9 @@ object TimestampFormatter {
   def apply(timeZone: TimeZone): TimestampFormatter = {
     apply(defaultPattern, timeZone, defaultLocale)
   }
+
+  val fractionPattern = "fraction-pattern"
```

Comment:
I'd put this up with the other vals

Reply:
I think it is better to move it to the private object `DateTimeFormatterHelper`